src/components/auction/FilledAuctionCard.tsx

```diff
@@ -0,0 +1,151 @@
+import { AuctionType, Pool, ScaledAuction } from '@blend-capital/blend-sdk';
+import { Box, Typography, useTheme } from '@mui/material';
+import { useMemo } from 'react';
+import { useBackstop, useHorizonAccount, usePoolOracle, useTokenBalance } from '../../hooks/api';
+import { calculateAuctionOracleProfit } from '../../utils/auction';
+import { toBalance, toCompactAddress } from '../../utils/formatter';
+import { DividerSection } from '../common/DividerSection';
+import { PoolComponentProps } from '../common/PoolComponentProps';
+import { Row } from '../common/Row';
+import { Section, SectionSize } from '../common/Section';
+import { StackedTextBox } from '../common/StackedTextBox';
+import { BidList } from './BidList';
+import { LotList } from './LotList';
+
+export interface FilledAuctionCardProps extends PoolComponentProps {
+  pool: Pool;
+  auction: ScaledAuction;
+  index: number;
+}
+
+export const FilledAuctionCard: React.FC<FilledAuctionCardProps> = ({ pool, auction, sx }) => {
+  const theme = useTheme();
+  const { data: poolOracle } = usePoolOracle(pool);
+  const { data: backstop } = useBackstop();
+  const { data: horizonAccount } = useHorizonAccount();
+
+  const assetKeys = useMemo(() => {
+    if (auction.type === AuctionType.Interest) {
+      const assets = new Set(auction.data.lot.keys());
+      for (const asset of Array.from(auction.data.bid.keys())) {
+        assets.add(asset);
+      }
+      return Array.from(assets);
+    }
+    return [];
+  }, [auction]);
+
+  const assetBalances = new Map<string, bigint>();
+  assetKeys.forEach((asset) =>
+    assetBalances.set(
+      asset,
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      useTokenBalance(asset, pool.reserves.get(asset)?.tokenMetadata.asset, horizonAccount).data ??
+        BigInt(0)
+    )
+  );
+
+  const auctionValue = useMemo(() => {
+    if (poolOracle && backstop) {
+      try {
+        return calculateAuctionOracleProfit(
+          auction.data,
+          auction.type,
+          pool,
+          poolOracle,
+          backstop.backstopToken
+        );
+      } catch (e) {
+        console.error('Error calculating auction value', e);
+      }
+    }
+    return undefined;
+  }, [auction, pool, poolOracle, backstop]);
```

Comment:
why are we showing an estimated oracle profits? this is historical info so estimating is not accurate based on current oracle prices